index.js

```diff
@@ -62,6 +62,10 @@ var supportLevel = (function () {
 		return 1;
 	}
 
+	if (process.env.TEAMCITY_VERSION) {
```

Comment:
This should take higher priority than env.TERM as it doesn't work even if it exports the right TERM. I think it should be below line 47.